client/components/token-field/index.jsx

```diff
@@ -181,6 +183,10 @@ var TokenField = React.createClass( {
 			onBlur: this._onBlur,
 		};
 
+		if ( value.length < 1 && placeholder ) {
```

Comment:
Minor: If there's only one value we'd expect to satisfy this condition (zero), do you think this might be clearer as `! value.length` or `value.length === 0` ?